src/client/components/pages/utils/getInstanceIdentifier.ts

```diff
@@ -1,5 +1,6 @@
 import { TERM } from 'common/constants';
 import CourseInstanceResponseDTO from 'common/dto/courses/CourseInstanceResponse';
+import { toTitleCase } from 'common/utils/util';
 
 interface semester {
```

Comment:
semester is a type,  so 's' would be capital like `interface Semester`.

Reply:
Great catch, Urji! Thanks for pointing this out. I fixed this in commit [4c53263](https://github.com/seas-computing/course-planner/pull/567/commits/4c5326350bdecec24514014571a3c4cd09d911b1)